src/styles/text.tsx

```diff
@@ -11,3 +15,30 @@ export const H2 = styled.h2`
   font-size: 1.5rem;
   margin: 0;
 `;
+
+export const H3 = styled.h3`
+  ${openSans.style}
+  font-size: 1.5rem;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+`;
+
+export const H4 = styled.h4`
+  color: ${COLORS.dark};
+  ${openSans.style}
+  font-size: 1rem;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+`;
+
+export const H4Centered = styled(H4)`
+  text-align: center;
+`;
+
+export const P = styled.p`
+  ${openSans.style}
+  font-size: 0.875rem;
+  font-weight: 400;
+`;
```

Comment:
Noticed Catherine added these in her PR and kinda needed them too, so I went ahead and included them here as well. I can deal w rebasing later just lmk!